src/components/Account/AccountBalances.tsx

```diff
@@ -112,7 +112,7 @@ export function MultipleBalances(props: MultipleBalancesProps) {
   ]
 
   return (
-    <>
+    <div style={{ maxWidth: "100%", whiteSpace: "nowrap", overflowX: "auto" }}>
```

Comment:
Waiting for Daniel. But this should only apply the mobile for now, not for desktop (I think the story wasn't too clear about that) 😉